### PR TITLE
Register page

### PR DESCRIPTION
**Pull request by miketvo**
This is not Thu's initial design, but I try to incorporate as much of her design philosophy as possible while maintaining our site-wide minimalism and utilizing existing code.

### CODE REVIEW

**Review by tnathu-ai — COMMENTED** [review]
+Missing profile picture (an upload file field)

File: public/mall/account/register/index.html [hunk 1]
```diff
@@ -67,8 +68,406 @@
...
+        <div class="register-field flex-container flex-align-items-center flex-justify-content-center">
+          <div class="register-item flex-container animated">
+            <label class="label" for="store-name">Store Name</label>
+            <input type="text" name="store_name" id="store-name" value="">
+          </div>
+        </div>
+
+        <div class="register-field flex-container flex-align-items-center flex-justify-content-center">
+          <div class="register-item flex-container animated">
+            <label class="label" for="store-category">Business Category</label>
```
Comment (doanyennhi) [inline public/mall/account/register/index.html:444]: label should be 'Store Category'

**Review by doanyennhi — CHANGES_REQUESTED** [review]
- Should add arrows for select elements
- Missing upload profile picture
- Fields have different lengths when screen size <= 950px

**Review by rider3458 — APPROVED** [review]

**Review by tnathu-ai — APPROVED** [review]
Well done!

**Pull request merged**